.github/ISSUE_TEMPLATE/user-facing-feature.md

```diff
@@ -1,40 +1,51 @@
 ---
 name: User facing feature
-about: For internal use
-title: ''
-labels: ''
+about: For internal use only; Open a user-facing feature tracking file
+title: "[FEATURE] - "
+labels: ["type: internal-feature", "user type: internal"]
 assignees: ''
 
 ---
 
-## What - The user facing feature being implemented
+## What
+<!-- What the end user will see -->
 
 ## Why
+<!-- The reason why this user-facing feature is being implemented -->
 
-## Personas - Who will this affect?
+## Personas
+<!-- Who will this affect? -->
 
-- [ ] SPO
-- [ ] Dapp Devs
+- [ ] SPOs
+- [ ] dApp Devs
 - [ ] Exchanges
-
-## Acceptance Criteria
+- [ ] Wallets
+- [ ] 3rd party tools
 
 ## Definition of done
 
-- [ ] Code review
+- [ ] Acceptance Criteria + User Stories & DoD created and singed-off (by PO, dev & test owners)
 - [ ] Builds successfully on CI
-- [ ] Includes documentation and/or examples for the functionality
+- [ ] Code & Test review (as per Acceptance Criteria)
+- [ ] There is documentation and/or examples for the new functionality (usage/response)
 - [ ] Log/record changes on **Vnext** (or similar depending on what we adopt)
-- [ ] Ticket number included in PR description
-- [ ] The new functionality is covered by dev/unit/property tests
-- [ ] Acceptance Criteria met
-- [ ] Test Engineering Sign-off + E2E automated tests
+- [ ] Ticket numbers included in PR description
+- [ ] All Acceptance Criteria met and covered by dev/unit/property/integration tests
+- [ ] Test Engineering Sign-off + System/E2E automated tests
+- [ ] Feature merged into master
```

Comment:
This is redundant if the issue is tagged in the PR's description. Github will render this on the issue.

Reply:
I propose keeping it as it is for better clarity.

Reply:
It's perfectly clear as it's shown on the the issue history automatically. This is busy work for my team.

Reply:
I am looking to make the DoD clear for as many people as possible, not only for developers, or only for a subteam. 

The DoD is a list of clear steps that need to happen before taking an action so avoiding any confusion is better than making assumptions in this case.